fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/type/NumberParmBehaviorUtil.java

```diff
@@ -0,0 +1,289 @@
+                    if (isInteger) {
+                        bindVariables.add(originalNumber);
+                        whereClauseSegment.append(tableAlias + DOT).append(NUMBER_VALUE);
+                        whereClauseSegment.append(JDBCOperator.EQ.value())
+                                .append(BIND_VAR);
+
+                    } else {
+                        // Not an Integer search. 
+                        // Bounds are based on precision.
+                        bindVariables.add(generateLowerBound(originalNumber));
+                        bindVariables.add(generateUpperBound(originalNumber));
+
+                        // <CODE>BASIC_NUMBER_VALUE > ? AND BASIC_NUMBER_VALUE <= ?</CODE> 
+                        whereClauseSegment.append(tableAlias + DOT).append(NUMBER_VALUE);
+                        whereClauseSegment.append(JDBCOperator.GT.value())
+                                .append(BIND_VAR);
+                        whereClauseSegment.append(JDBCConstants.SPACE).append(JDBCOperator.AND)
+                                .append(JDBCConstants.SPACE);
+                        whereClauseSegment.append(tableAlias + DOT).append(NUMBER_VALUE);
+                        whereClauseSegment.append(JDBCOperator.LTE.value())
+                                .append(BIND_VAR);
+                    }
+
+                    break;
+                case NE:
+                    // ne - not equals is a specific not in range search.
+                    if (isInteger) {
+                        bindVariables.add(originalNumber);
+                        whereClauseSegment.append(tableAlias + DOT).append(NUMBER_VALUE);
+                        whereClauseSegment.append(JDBCOperator.NE.value())
+                                .append(BIND_VAR);
+                    } else {
+                        // Bounds are based on precision.
+
+                        bindVariables.add(generateLowerBound(originalNumber));
+                        bindVariables.add(generateUpperBound(originalNumber));
+
+                        // <CODE>BASIC_NUMBER_VALUE <= ? OR BASIC_NUMBER_VALUE > ?</CODE> 
+                        whereClauseSegment.append(tableAlias + DOT).append(NUMBER_VALUE);
+                        whereClauseSegment.append(JDBCOperator.LTE.value())
+                                .append(BIND_VAR);
+                        whereClauseSegment.append(JDBCConstants.SPACE).append(JDBCOperator.OR)
+                                .append(JDBCConstants.SPACE);
+                        whereClauseSegment.append(tableAlias + DOT).append(NUMBER_VALUE);
+                        whereClauseSegment.append(JDBCOperator.GT.value())
+                                .append(BIND_VAR);
+                    }
+                    break;
+                case AP:
+                    // ap - approximate
+                    // -10%
+                    bindVariables.add(originalNumber.multiply(LOWER_BOUND));
```

Comment:
definitely edge case, and probably doesn't matter, but should this be `-10%` of the start of the implicit range instead of `-10%` of the originalNumber?

Reply:
I understood it as -/+10% of the original 

```
Note that the recommended value for the approximation is 10% of the stated value (or for a date, 10% of the gap between now and the date), but systems may choose other values where appropriate
```

Reply:
Discussed with Lee - OK for now.